oss_fuzz_integration/runner.py

```diff
@@ -62,6 +65,30 @@ def download_full_public_corpus(project_name, target_corpus_dir: None):
         subprocess.check_call(f"unzip {target_zip} -d {target_fuzzer_dir}/", shell=True)
 
 
+def prepare_introspector(curr_dir):
```

Comment:
I think we should avoid this. Is there a reason for not putting it in the diff and handling it like we do with other languages?